addons/optionsmenu/stringtable.xml

```diff
@@ -323,20 +323,24 @@
             <Polish>Debuguj do schowka</Polish>
             <Spanish>Depurar al portapapeles</Spanish>
             <Czech>Debug do schránky</Czech>
+            <German>Debug in die Zwischenablage</German>
         </Key>
         <Key ID="STR_ACE_OptionsMenu_DumpDebugTooltip">
             <English>Sends debug information to RPT and clipboard.</English>
             <Polish>Wysyła informacje o debugowaniu do RPT oraz schowka.</Polish>
             <Spanish>Envía información de depuración al RPT y el portapapeles.</Spanish>
             <Czech>Pošle debug informace do RPT a schránky.</Czech>
+            <German>Sendet Debug-Informationen ans RPT und in die Zwischenablage.</German>
```

Comment:
Klingt besser. Allerdings ist protokollieren dann nicht mehr bei der Zwischenablage korrekt, da man in der Zwischenablage nichts protokolliert. Ich habe es mal zu "Protokolliert Debug-Informationen im RPT und speichert sie in der Zwischenablage." gewmacht. Die Frage wäre auch wofür RPT steht. Der Artikel wäre interessant. Ich bin der Meinung, dass es "der RPT" ist.